src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,12 @@
+package wolox.training.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import wolox.training.models.Book;
+
+public interface BookRepository extends JpaRepository<Book, Integer> {
```

Comment:
We should use the `Long` data type for the id attribute
```suggestion
public interface BookRepository extends JpaRepository<Book, Long> {
```